src/wh_comm.c

```diff
@@ -330,6 +331,22 @@ int wh_CommServer_SendResponse(whCommServer* context,
     return rc;
 }
 
+int wh_CommClient_Cancel(whCommClient* context, uint16_t magic)
+{
+    int rc = WH_ERROR_NOTREADY;
+    if (context == NULL || context->transport_cb == NULL ||
+        context->transport_cb->Cancel == NULL) {
+        return WH_ERROR_BADARGS;
+    }
+    context->hdr->magic = magic;
+    context->hdr->kind = wh_Translate16(magic,
+        WH_MESSAGE_KIND(WH_MESSAGE_GROUP_CANCEL, 0));
```

Comment:
Should we make actions for the cancel message group? Even if there is just one for now, probably a good idea.